index.css

```diff
@@ -0,0 +1,142 @@
+body {
```

Comment:
- [ Optional ] The `font-family` used on the Figma design is `Poppins` but if you look keenly at your design the `font` does not resemble the one on the `Figma` template. This is because you have not imported the `Poppins` font file. You can do this by adding the following `CSS` rule to the top of your `index.css` file

```CSS
  @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```
## Figma template
<img src="https://user-images.githubusercontent.com/16392046/219853011-8af7204e-1102-4277-b943-93aec073934b.png" width="375px" />



## Your design
<img src="https://user-images.githubusercontent.com/16392046/219852978-9fa8d5fc-3672-4cd6-8540-598c92cd3e23.png" width="375px" />